{"identifier":{"interfaceLanguage":"occ","url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult\/failureReason"},"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult"]]},"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"@property"},{"kind":"text","text":" ("},{"kind":"keyword","text":"nonatomic"},{"kind":"text","text":", "},{"kind":"keyword","text":"strong"},{"kind":"text","text":", "},{"kind":"keyword","text":"readonly"},{"kind":"text","text":") "},{"kind":"typeIdentifier","preciseIdentifier":"c:objc(cs)NSString","text":"NSString"},{"kind":"text","text":" * "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":";"}],"languages":["occ"]}]}],"metadata":{"modules":[{"name":"SMIClientCore"}],"fragments":[{"kind":"identifier","text":"failureReason"}],"title":"failureReason","symbolKind":"property","navigatorTitle":[{"kind":"identifier","text":"failureReason"}],"externalID":"c:objc(pl)SMIRoutingResult(py)failureReason","roleHeading":"Instance Property","role":"symbol"},"abstract":[{"type":"text","text":"The reason for the routing failure"}],"variants":[{"paths":["\/documentation\/smiclientcore\/smiroutingresult\/failurereason"],"traits":[{"interfaceLanguage":"occ"}]}],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIRoutingResult":{"url":"\/documentation\/smiclientcore\/smiroutingresult","abstract":[{"text":"Represents a routing result within an ","type":"text"},{"code":"SMIConversationEntry","type":"codeVoice"},{"text":".","type":"text"}],"kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult","title":"SMIRoutingResult","role":"symbol","type":"topic","fragments":[{"kind":"identifier","text":"SMIRoutingResult"}]},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"title":"SMIClientCore","abstract":[],"url":"\/documentation\/smiclientcore","role":"collection","type":"topic","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","kind":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIRoutingResult/failureReason":{"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIRoutingResult\/failureReason","fragments":[{"kind":"identifier","text":"failureReason"}],"kind":"symbol","title":"failureReason","abstract":[{"type":"text","text":"The reason for the routing failure"}],"role":"symbol","type":"topic","url":"\/documentation\/smiclientcore\/smiroutingresult\/failurereason"}}}